docs/api/sql/Function.md

```diff
@@ -3266,6 +3266,46 @@ Output:
 LINESTRING(0 0, 1 0)
 ```
 
+## ST_RemoveRepeatedPoints
+
+Introduction: This function eliminates consecutive duplicate points within a geometry, preserving endpoints of LineStrings. It operates on (Multi)LineStrings, (Multi)Polygons, and MultiPoints, processing GeometryCollection elements individually. When an optional 'tolerance' value is provided, vertices within that distance are also considered duplicates.
+
+Format:
+
+`ST_RemoveRepeatedPoints(geom: Geometry, tolerance: Double)`
+
+`ST_RemoveRepeatedPoints(geom: Geometry)`
+
+Since: `v1.7.0`
+
+SQL Example:
+
+```sql
```

Comment:
Can you provide very detailed examples for this function? Similar to the PostGIS examples